### DISCUSSION TITLE
What option should I use for building bridge?

### DISCUSSION THREAD

**Discussion by chlsu** [mitmproxy/mitmproxy#6330] [Q&A]
I want to apply mitmproxy to bridge(transparent equipment) to inspect HTTP, HTTPS url.
But I can't find any option i could use to direct from mitmproxy to bridge.

i made bridge0. I applied next code, but It doesn't work.

iptables -t nat -A PREROUTING -i bridge0 -p tcp --dport 80 -j REDIRECT --to-port 8080
iptables -t nat -A PREROUTING -i bridge0 -p tcp --dport 443 -j REDIRECT --to-port 8080
ip6tables -t nat -A PREROUTING -i bridge0 -p tcp --dport 80 -j REDIRECT --to-port 8080
ip6tables -t nat -A PREROUTING -i bridge0 -p tcp --dport 443 -j REDIRECT --to-port 8080

**Comment by ThomasBucaioni** [comment]
You're working with VMs? Maybe the simpler would be make a router VM, run `mitmproxy` on this VM, and route the traffic of the other VMs through the router
